connector_search_engine/models/se_index.py

```diff
@@ -64,11 +80,12 @@ def _jobify_batch_export(self):
         description = _("Prepare a batch export of index '%s'") % self.name
         self.with_delay(description=description).batch_export()
 
-    @api.model
+    @api.multi
```

Comment:
@simahawk Why `api.multi`? We don't use fields of self... The method is callable on the model

Reply:
actually it acts on the current record and we use self.name for instance

Reply:
just realized this action + view was removed in v10 and I missed it during rebase... nothing to review anymore here :)